.gitbook.yml

```
@@ -1,5 +1,5 @@
 root: ./docs
 
 structure:
-  readme: ../README.md
-  summary: SUMMARY.md
+  readme: README.md
+  summary: SUMMARY.md
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)